File: Examples/CSharp/Working-with-Print/ByDefaultPrinter.cs
using Aspose.Diagram;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Diagram.Examples.CSharp.Working_with_Print
{
    public class ByDefaultPrinter
    {
        public static void Run() 
        {
            // ExStart:ByDefaultPrinter
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Print();

            // Load source Visio diagram
            Diagram diagram = new Diagram(dataDir + "Drawing1.vsdx");

            // Call the print method to print whole Diagram using the default printer
            diagram.Print();
            // ExEnd:ByDefaultPrinter
        }
    }
}

File: Examples/VisualBasic/Working-with-Print/ByDefaultPrinter.vb
Imports Aspose.Diagram
Imports System
Public Class ByDefaultPrinter
    Public Shared Sub Run()
        ' ExStart:ByDefaultPrinter
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_Print()

        ' Load source Visio diagram
        Dim diagram As New Diagram(dataDir & Convert.ToString("Drawing1.vsdx"))

        ' Call the print method to print whole Diagram using the default printer
        diagram.Print()
        ' ExEnd:ByDefaultPrinter
    End Sub
End Class
